docs/visual-basic/getting-started/additional-resources.md

```diff
@@ -1,88 +1,75 @@
+|[Visual Basic IDE](https://social.msdn.microsoft.com/Forums/en-US/home?forum=vbide)|Discusses working in the [!INCLUDE[vsprvs](~/includes/vsprvs-md.md)] integrated development environment.|
+|[Visual Basic General](https://social.msdn.microsoft.com/Forums/vstudio/en-US/home?forum=vbgeneral)|Discusses issues with [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] that are not discussed in other forums.|
+
+### Videos and webcasts
+
+|Term|Definition|
+|----------|----------------|
+|[Channel9](https://channel9.msdn.com/)|Provides continuous community through videos, Wikis, and forums.|
+|[I'm a VB](https://channel9.msdn.com/Tags/vb+team)|Watch these interviews and discover what the experts love about Visual Basic and Visual Studio.|
+|[Microsoft Virtual Academy](https://mva.microsoft.com/)|Free Microsoft training delivered by experts.|
   
 ### Support  
-  
+
 |Term|Definition|  
 |----------|----------------|  
-|[Microsoft Help and Support](http://go.microsoft.com/fwlink/?LinkID=108287)|Provides access to Knowledge Base (KB) articles, downloads and updates, support webcasts, and other services.|  
-|[Microsoft Connect](http://connect.microsoft.com/)|Enables you to file bugs or provide suggestions to Microsoft about Visual Studio. You can also report a bug by choosing **Report a Bug** on the **Help** menu.|  
-  
-## Third-Party Resources  
- The MSDN web site provides information on current third-party sites and newsgroups of interest. For the most current list of resources available, see the [MSDN Visual Basic Community Web](http://go.microsoft.com/fwlink/?LinkID=77372) site.  
-  
+|[Microsoft Support](https://support.microsoft.com)|Provides access to Knowledge Base (KB) articles, downloads and updates, support webcasts, and other services.|  
+|[Visual Studio Questions](https://developercommunity.visualstudio.com)|Enables you to file bugs or provide suggestions to Microsoft about .NET and Visual Studio. You can also report a bug by selecting **Help** > **Send Feedback** > **Report a Problem** in Visual Studio.|  
+
+## Third-party resources  
+ 
 |Term|Definition|  
 |----------|----------------|  
-|[DevX Visual Basic Zone](http://go.microsoft.com/fwlink/?LinkId=145978)|Provides in-depth technical articles for today's Visual Basic developer.|  
-|[vb.dotnet.technical](http://go.microsoft.com/fwlink/?LinkId=145986)|Provides a forum for discussion of new features in [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] in the DevX forums.|  
-  
-## See Also  
- [Getting Started](../../visual-basic/getting-started/index.md)  
- [Talk to Us](/visualstudio/ide/talk-to-us)
+|[DevX Visual Basic Zone](http://www.devx.com/vb)|Provides in-depth technical articles for today's Visual Basic developer.|
+|[vb.dotnet.technical](http://go.microsoft.com/fwlink/?LinkId=145986)|Provides a forum for discussion of new features in [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] in the DevX forums.|
```

Comment:
This ultimately redirectly to devx. An alternative link is http://www.vbcity.com, which is an active VB.NET site.